src/sentry/integrations/github/client.py

```diff
@@ -189,7 +190,7 @@ def is_error_fatal(self, error: Exception) -> bool:
         return super().is_error_fatal(error)
 
 
-class GitHubBaseClient(GithubProxyClient, RepositoryClient, CommitContextClient):
+class GitHubBaseClient(GithubProxyClient, RepositoryClient, CommitContextClient, RepoTreesClient):
```

Comment:
`RepoTreesClient` will have all the logic to get all the files from all repositories.
I will leave the original code in this client since we will roll over to the new code path.
After we're fully migrated we will delete the functions mentioned here.

Reply:
I think so 😅 We may want to meet to go over it.